dodder-common/src/test/java/cc/dodder/common/util/StringUtilTest.java

```diff
@@ -0,0 +1,80 @@
+package cc.dodder.common.util;
+
+import cc.dodder.common.entity.Node;
+import cc.dodder.common.entity.Torrent;
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class StringUtilTest {
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testByte2HexStr() {
+        assertEquals("060200", StringUtil.byte2HexStr(new byte[]{ (byte)6, (byte)2, (byte)0 }));
+    }
+
+    @Test
+    public void testStr2Unicode() throws Exception {
+        assertEquals("//u005c", StringUtil.str2Unicode("\\"));
+        assertEquals("//u0066//u006f//u006f", StringUtil.str2Unicode("foo"));
+    }
+
+    @Test
+    public void testFormatSize() {
+        assertEquals("512 GB", StringUtil.formatSize(0x1p+39));
+        assertEquals("16 MB", StringUtil.formatSize(0x1p+24));
+        assertEquals("1024 KB", StringUtil.formatSize(0x1p+20));
+        assertEquals("4 B", StringUtil.formatSize(0x1p+72));
```

Comment:
Are these two tests both needed for coverage? (it looks to me like they test the same lines of code)

> assertEquals("4 B", StringUtil.formatSize(0x1p+72));
> assertEquals("0 B", StringUtil.formatSize(0.0));